lib/Doctrine/Common/Annotations/DocParser.php

```diff
@@ -1456,4 +1458,102 @@ private function resolvePositionalValues(array $arguments, string $name): array
 
         return $values;
     }
+
+    /**
+     * Try to instantiate the annotation and catch and process any exceptions related to failure
+     *
+     * @param class-string             $name
+     * @param array<string,mixed>|null $arguments
+     *
+     * @return object
+     *
+     * @throws AnnotationException
+     */
+    private function instantiateAnnotiation(string $originalName, string $context, string $name, ?array $arguments = null)
+    {
+        try {
+            if ($arguments === null) {
```

Comment:
couldn't you use an empty array as default value and then always use `return new $name(...$arguments);` ?